src/main/java/me/guichaguri/tickratechanger/TickrateContainer.java

```diff
@@ -46,6 +46,7 @@
 
     @EventHandler
     public void preInit(FMLPreInitializationEvent event) {
+		new TickrateChanger();

```

Comment:
In `TickrateChanger` you have a static variable called `INSTANCE` which is initialized in the constructor. The constructor is never called so `INSTANCE `is never initialized but it still used, throwing a `NullPointerException`. If I am missing something somewhere or FML/Forge automatically calls the class based on the two interfaces it implements let me know and I can remove it, but when I ran it without initializing `TickrateChanger` it crashed and when I added that line it ran and the tickrates changed properly.